apps/frontend/src/shared/i18n/locales/en/settings.json

```
@@ -183,7 +186,14 @@
     "fineTuneDescription": "Adjust from 75% to 200% in 5% increments",
     "default": "Default",
     "comfortable": "Comfortable",
-    "large": "Large"
+    "large": "Large",
+    "apply": "Apply"
+  },
+  "theme": {
```

Biome (2.1.2)

[error] The key theme was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)